webapp/sources/rudder/rudder-web/src/main/webapp/templates-hidden/components/ComponentNodeProperties.html

```diff
@@ -102,12 +102,12 @@
                      <small class="text-danger" ng-if="!editedProperties[property.name].new.isValid">JSON check is enabled, but the value format is invalid</small>
                    </div>
                  </td>
-                 <td class="text-center" ng-class="{'default-actions':!isEdited(property.name), 'edit-actions is-edited':isEdited(property.name)}">
-                   <div ng-if="!isEdited(property.name) && property.rights !== 'read-only'">
```

Comment:
you should not remove this, it's to prevent edition of inventory/datasources properties that cannot be modified